sample/marchingSquares.glsl

```diff
@@ -0,0 +1,204 @@
+#include "../generative/pnoise.glsl"
+#include "../math/const.glsl"
+#include "../math/rotate2d.glsl"
+#include "../sample.glsl"
+
+/*
+Author: Guido Schmidt
```

Comment:
the field should be `original_author:  Guido Schmidt`

Reply:
Oh yes, I struggled with this, too. I've went with `sample` because I felt the algorithm is kind of resampling a given input to create isolines. `sdf` could also be a good option (though I'm using it for non-sdf related things, too) 🤔

Reply:
it's a tricky one, right? I think by sampling a texture by default we make it have sense here. I'm glad you were also thinking on that and I'm not pushing you in any direction you don't want to go

Reply:
Oh right, I was taking the `sample/2DCube` as a reference, it has `Author`, will change 👍🏽

Reply:
Oh! you are right! Good catch! My bad, I need to fix it https://github.com/patriciogonzalezvivo/lygia/blob/main/sample/3DSdf.glsl#L7 
I love that I put capital `A` like an egomaniac lol